.github/workflows/examples.yml

```diff
@@ -30,6 +30,8 @@ jobs:
         echo "::set-output name=BEFORE::$(git status --porcelain -b)"
     - name: Run tests
       run: |
-        python examples/example_tabular_classification.py
-        python examples/example_tabular_regression.py
+        python examples/tabular/20_basics/example_tabular_classification.py
```

Comment:
Can you make sure that the documentation also see this change?
For example, we have to change this [line](https://github.com/automl/Auto-PyTorch/blob/b5d1c8b0096c08e4b827ebfcc2a951eb70f0e446/docs/conf.py#L71).

Reply:
okay